strongbox-storage/strongbox-storage-core/src/main/java/org/carlspring/strongbox/configuration/MutableConfiguration.java

```diff
@@ -114,6 +117,14 @@ public void setPort(int port)
         this.port = port;
     }
 
+    public Set<String> getDigestAlgorithmSet() {
+        return digestAlgorithmSet;
+    }
+
+    public void setDigestAlgorithmSet(Set<String> digestAlgorithmSet) {
+        this.digestAlgorithmSet = digestAlgorithmSet;
+    }
```

Comment:
Cosmetic: Braces on new lines.

```suggestion
    public Set<String> getDigestAlgorithmSet()
    {
        return digestAlgorithmSet;
    }
    
    public void setDigestAlgorithmSet(Set<String> digestAlgorithmSet)
    {
        this.digestAlgorithmSet = digestAlgorithmSet;
    }
```